server/channels/app/platform/metrics.go

```diff
@@ -56,6 +62,12 @@ func (ps *PlatformService) resetMetrics() error {
 		cfgFn:       ps.Config,
 		metricsImpl: ps.metricsIFace,
 		logger:      ps.logger,
+		getPluginsEnv: func() *plugin.Environment {
+			if ps.pluginEnv == nil {
+				return nil
+			}
+			return ps.pluginEnv.GetPluginsEnvironment()
+		},
```

Comment:
could it be shortened to `getPluginsEnv: ps.pluginEnv.GetPluginsEnvironment`?

Reply:
Yeah, should be safe.

Reply:
Okay, now I know why I made it that way in the first place :) Tests are crashing due to `nil` dereference.

Reply:
Given `ps.pluginEnv` can be `nil` I'll revert.